rasa/core/policies/ted_policy.py

```diff
@@ -156,7 +164,7 @@ class TEDPolicy(Policy):
     """
 
     # please make sure to update the docs when changing a default parameter
-    defaults = {
+    default_config = {
```

Comment:
💡 all `GraphComponents` have a `default_config`.

Reply:
Would it make sense to switch to something like Hydra for the configs in the long term or is there any reason we cannot do that? (would give us lots of type and consistency checks for free and should make hierarchical configs easier to manage)